docs/DEVELOPER-GUIDE.md

```diff
@@ -0,0 +1,38 @@
+# Developer Guide
+This document describes how to create a pull request for contributing to Appsody along with various ways to test code changes. 
+
+## create pull requests:
+This section describes how to create a pull request.
+* create a fork for the appsody repo, ex:
+    * https://github.com/appsody/appsody
+        * click on the Fork button
+* using git, clone your fork and create a unique branch for each issue or code task
+* after committing and pushing your changes use the appsody github webpage to create a pull request
+    * the github webpage for the upstream repo should show an indicator after pushing your changes, click the "Compare & pull request" button
+![image info](pull-request.png)
+
+* if you would like your code changes to be known/tracked but are not ready for them to be reviewed/approved you can create a draft pull request using the pull-down
+![image info](draft-pull-request.png)
+
+## testing:
+There are various ways to test your own code or someone else’s pull request (or branch):
+* test your code changes against the test bucket
+    * make test
+* build the appsody binary files for the various platforms
+    * make build-darwin (mac)
+    * make build-windows (windows)
+    * make build-linux (linux)
+        * this creates an apposdy binary in the build dir of the repo
+* test appsody binary file
```

Comment:
This looks more like a command than a title, if it's meant to be a section title it needs to be something like "test the local appsody binary file"